API/.env.template

```
@@ -3,7 +3,7 @@ DJANGO_DEBUG=
 DJANGO_ALLOWED_HOSTS=
 DJANGO_ENV=
 DB_NAME=FabLab
-DB_USER=root
-DB_PASSWORD=RootPass123
+DB_USER=fablab_user
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DB_USER key should go before the DJANGO_ALLOWED_HOSTS key

(UnorderedKey)